### PR TITLE
[3-6] Adding translation to project. Refactored code.

### PR DESCRIPTION
**Pull request by GeranSkot**
[3-6] Adding translation to project. Refactored code.

### CODE REVIEW

File: exalearn-ui/src/app/app.module.ts [hunk 1]
```diff
@@ -5,10 +5,33 @@ import { MainPageModule } from './pages/main-page/main-page.module';
...
 
 @NgModule({
 	declarations: [AppComponent],
-	imports: [BrowserModule, AppRoutingModule, MainPageModule, LoginPageModule],
+	imports: [
+		BrowserModule,
+		AppRoutingModule,
+		HttpClientModule,
+		TranslateModule.forRoot({
+			loader: {
```
Comment (YanaBr) [inline exalearn-ui/src/app/app.module.ts]: Maybe create above const object for translate module and then put it inside forRoot(..)

File: exalearn-ui/src/app/components/footer/footer.component.ts [hunk 2]
```diff
@@ -1,10 +1,24 @@
...
 	templateUrl: './footer.component.html',
 	styleUrls: ['./footer.component.scss']
 })
 export class FooterComponent implements OnInit {
-	ngOnInit(): void {}
+	//TODO TESTING TRANSLATION MODULE
+	translate: TranslateService;
+
+	constructor(public translateService: TranslateService) {
+		translateService.addLangs(['en_EN', 'ru_RU']);
```
Comment (YanaBr) [inline exalearn-ui/src/app/components/footer/footer.component.ts]: I think we can create const object for list of languages

Comment (vstrabko) [inline exalearn-ui/src/app/components/footer/footer.component.ts]: I think we should keep langs in separate file and import it here.

File: exalearn-ui/src/app/pages/test-page/test-page.component.ts [hunk 3]
```diff
@@ -10,9 +12,11 @@ import { Question } from '../../interfaces/interfaces';
 export class TestPageComponent implements OnInit {
 	testQuestions: Question[] = [];
 
-	constructor() {}
+	constructor(public translateService: TranslateService) {}
 
 	ngOnInit() {
 		this.testQuestions = questions;
+		//TODO use user state from backend
```
Comment (vstrabko) [inline exalearn-ui/src/app/pages/test-page/test-page.component.ts]: Expand a comment a bit. Describe in short what can happen if we remove this line.

File: exalearn-ui/src/app/components/footer/footer.component.ts (hunk 2 above)
Comment (vstrabko) [reply exalearn-ui/src/app/components/footer/footer.component.ts]: You have locales specified down in the code. Maybe we can use them for now?

**Review by vstrabko — CHANGES_REQUESTED** [review]

**Review by vstrabko — APPROVED** [review]

**Review by YanaBr — APPROVED** [review]

**Pull request merged**